core/domain/app_feedback_report_domain_test.py

```diff
@@ -1526,13 +1526,13 @@ class AppFeedbackReportFilterDomainTests(test_utils.GenericTestBase):
     def setUp(self) -> None:
         super().setUp()
         self.filter = app_feedback_report_domain.AppFeedbackReportFilter(
-            app_feedback_report_constants.FILTER_FIELD_NAMES.platform,
+            app_feedback_report_constants.FILTER_FIELD_NAMES.PLATFORM,
             ['web', 'android'])
 
     def test_to_dict(self) -> None:
         app_feedback_report_constants.PLATFORM_CHOICES.sort()
         expected_dict = {
-            'filter_field': 'platform',
+            'filter_field': 'PLATFORM',
```

Comment:
Shouldn't the constant value remain unchanged since only the Python names are changing?

Reply:
Good catch I think I accidentally changed this value

Reply:
Done